### PR TITLE
Dryrun tags bug fixing

### PR DESCRIPTION
**Pull request by blackdog0403**
Should merge after #492 merged into master.

Changing 'dryrun' -> 'config_dryrun' will cause conflicts and bugs

### CODE REVIEW

**Comment by blackdog0403** [comment]
I re-change 'config_dryrun' to 'dryrun'. Let's discuss about it then go!

File: ansible/roles/kraken.fabric/kraken.fabric.flannel/tasks/main.yaml [hunk 1]
```diff
@@ -107,7 +107,7 @@
   command: >
     {{ kubectl }} --kubeconfig={{ kubeconfig | expanduser }} apply -f {{ config_base | expanduser }}/{{ cluster.name }}/auth/rbac-canal-policy.yaml
   register: job_rbac_canal_results
-  when: 
+  when:
     - "cluster.kubeAuth.authz.rbac is defined"
     - "(not (dryrun | bool))"
```
Comment (joejulian) [inline ansible/roles/kraken.fabric/kraken.fabric.flannel/tasks/main.yaml]: In most places you don't place quotes around the when clauses. Let's be consistent and not put quotes here.

**Comment by joejulian** [comment]
Well, it does look good, but dryrun does not work, so somehow the logic is failing.

**Comment by blackdog0403** [comment]
@joejulian there is some error on here which is very weird to me. I'm working on it.

```
- name: Wait for api server
  wait_for:
    host: "{{ item | regex_replace('https://','') }}"
    port: 443
    timeout: 600
  with_items: "{{ api_servers }}"
  vars:
    kubefn: "{{ config_base | expanduser }}/{{ cluster.name }}/admin.kubeconfig"
    kubeconfig: "{{ kubefn | is_file | ternary( kubefn, '/dev/null' ) }}"
    api_servers: "{{ lookup('file', kubeconfig) | from_yaml | json_query('clusters[*].cluster.server') }}"
  when: not ( dryrun | bool )
```
when we run this with dryrun admin.kubeconfig file is not generated for intention.
However, even if dryrun tag is set, they check the existency of admin.kubeconfig file

**Pull request merged**